packages/amplify-provider-awscloudformation/src/root-stack-builder/root-stack-builder.ts

```diff
@@ -140,51 +173,67 @@ export class AmplifyRootStack extends cdk.Stack implements AmplifyRootStackTempl
         ],
       },
     });
-  };
+  }
 
   // add Function for Custom Resource in Root stack
   /**
-   *
-   * @param _
-   * @returns
+   * Synthesizes the template into a JSON string
    */
-  public renderCloudFormationTemplate = (_: ISynthesisSession): string => {
-    return JSON.stringify(this._toCloudFormation(), undefined, 2);
-  };
+  public renderCloudFormationTemplate = (_: ISynthesisSession): string => JSON.stringify(this._toCloudFormation(), undefined, 2);
 }
 
 /**
- * additional class to merge CFN parameters and CFN outputs as cdk doesnt allow same logical ID of constructs in same stack
+ * Additional class to merge CFN parameters and CFN outputs as cdk doesn't allow same logical ID of constructs in same stack
  */
 export class AmplifyRootStackOutputs extends cdk.Stack implements AmplifyRootStackTemplate {
-  constructor(scope: cdk.Construct, id: string, props: AmplifyRootStackProps) {
-    super(scope, id, props);
-  }
   deploymentBucket?: s3.CfnBucket;
   authRole?: iam.CfnRole;
   unauthRole?: iam.CfnRole;
 
-  addCfnParameter(props: cdk.CfnParameterProps, logicalId: string): void {
+  /**
+   * Method not implemented
+   */
+  // eslint-disable-next-line class-methods-use-this
+  addCfnParameter(/* _props: cdk.CfnConditionProps, _logicalId: string */): void {
     throw new Error('Method not implemented.');
   }
+
+  /**
+   * Adds an output to the stack
+   */
   addCfnOutput(props: cdk.CfnOutputProps, logicalId: string): void {
     try {
+      // eslint-disable-next-line no-new
       new cdk.CfnOutput(this, logicalId, props);
     } catch (error) {
       throw new Error(error);
     }
   }
-  addCfnMapping(props: cdk.CfnMappingProps, logicalId: string): void {
+
+  /**
+   * Method not implemented
+   */
+  // eslint-disable-next-line class-methods-use-this
+  addCfnMapping(): void {
     throw new Error('Method not implemented.');
   }
-  addCfnCondition(props: cdk.CfnConditionProps, logicalId: string): void {
+
+  /**
+   * Method not implemented
+   */
+  // eslint-disable-next-line class-methods-use-this
+  addCfnCondition(/* _props: cdk.CfnConditionProps, _logicalId: string */): void {
```

Comment:
was this added intentionally ?

Reply:
this was to get the linter to stop complaining about unused parameters but still show what the parameters should be if/when this method needs to be implemented